modules/table/src/integrationTest/java/org/apache/ignite/distributed/ITTablePersistenceTest.java

```diff
@@ -125,14 +130,18 @@
 
     /** {@inheritDoc} */
     @Override public Path getListenerPersistencePath(PartitionListener listener) {
-        return ((RocksDbStorage) listener.getStorage()).getDbPath();
+        return paths.get(listener);
     }
 
     /** {@inheritDoc} */
     @Override public RaftGroupListener createListener(Path workDir) {
-        return new PartitionListener(
-            new RocksDbStorage(workDir.resolve(UUID.randomUUID().toString()), ByteBuffer::compareTo)
-        );
+        Path tableDir = workDir.resolve(UUID.randomUUID().toString());
```

Comment:
Can we update the parent class, so that we can get rid of this fake directory?

Reply:
I think I can move it to the parent class. Should I move a set of paths as well?